src/core/reducers/data/personalization.ts

```diff
@@ -0,0 +1,57 @@
+import { createTransform, persistReducer } from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import Actions from '../../actions';
+import Adapter from '../../adapters/personalization';
+import Selectors from '../../selectors';
+import Store from '../../store';
+
+export type Action = Actions.UpdateBiasing;
+export type State = Store.Personalization;
+
+export const STORAGE_KEY = 'gb-personalization';
+export const STORAGE_WHITELIST = ['biasing'];
+
+export const DEFAULT: State = {
+  biasing: {
+    allIds: [],
+    byId: {},
+  }
+};
+
+function updatePersonalization(state: State = DEFAULT, action: Action): State {
+  switch (action.type) {
+    case Actions.UPDATE_BIASING: return updateBiasing(state, action.payload);
+    default: return state;
+  }
+}
+
+export const updateBiasing = (state: State, payload: Actions.Payload.Personalization.Biasing) => {
+  const byId = {
+    ...state.biasing.byId,
+    [payload.variant]: {
+      ...state.biasing.byId[payload.variant],
+      [payload.key]: payload.bias
+    }
+  };
+
+  const allIds = Adapter.pruneBiases(insertSorted(state.biasing.allIds, { variant: payload.variant, key: payload.key }),
```

Comment:
not sure if the comment got removed somehow but I don't want to do the sorting before we put the data in the store, only when selecting it out of the store, this also means we don't need to pass the `config` with this action